tests/src/ut_control_common.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2023 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Standard Libraries */
+#include <stdio.h>
+#include <stdlib.h>
+
+/* Module Includes */
+#include <ut_log.h>
+
+typedef struct
+{
+    char *buffer;
```

Comment:
your calling everything memory, then call your pointer buffer,

Reply:
What is your suggestion here?